resources/js/components/ui/command-menu.tsx

```
@@ -100,7 +100,7 @@ const CommandMenu = ({
           <Modal
             className={cx(
               "row-start-2 bg-popover text-start text-popover-foreground shadow-lg outline-none ring ring-muted-foreground/15 md:row-start-1 dark:ring-border",
-              "max-h-[calc(var(--visual-viewport-height)*0.8)] w-full sm:fixed sm:top-[10%] sm:left-1/2 sm:-translate-x-1/2",
+              "sm:-translate-x-1/2 max-h-[calc(var(--visual-viewport-height)*0.8)] w-full sm:fixed sm:top-[10%] sm:left-1/2",
```

Biome (2.4.13)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)